### DISCUSSION TITLE
Edit form: default value for Selectfield

### DISCUSSION THREAD

**Discussion by SergShulga** [aminalaee/sqladmin#651] [Q&A]
Hi all!  I have redefined one of the form fields as a SelectField with a set of values ​​(I want to limit the user to the available options).  

Everything works fine, but there is a problem: when you go to edit an existing record, the value that this field had from the database disappears.  It becomes either the first one in the list or empty if I use allow_blank=True.
  
Question: how can I fill this field by default with the value that was previously saved (in this case the field has a valid value - earlier I also selected it from the list)?

**Comment by SergShulga** [comment]
Resolved

**Reply by aminalaee** [reply]
Please post your solution as it might help other people.
